drivers/rilmodem/sim.c

```diff
@@ -640,6 +641,59 @@ static void configure_active_app(struct sim_data *sd,
 	}
 }
 
+static void sim_send_set_uicc_subscription(struct ofono_sim *sim, int slot_id, int app_index, int sub_id, int sub_status)
+{
+	struct sim_data *sd = ofono_sim_get_data(sim);
+	struct parcel rilp;
+
+	DBG("");
+
+	g_ril_request_set_uicc_subscription(sd->ril, slot_id, app_index,
+						sub_id, sub_status, &rilp);
+	g_ril_send(sd->ril, QCOM_MSIM_RIL_REQUEST_SET_UICC_SUBSCRIPTION, &rilp,
+		   NULL, NULL, NULL);
```

Comment:
I would add here a callback function to check for message->error and call g_ril_print_response_no_args() (see for instance mtkmodem/voicecall.c:mtk_set_indication_cb(), so we know when the call failed.